firstradeAPI.py

```diff
@@ -41,10 +41,11 @@ def firstrade_init(FIRSTRADE_EXTERNAL=None):
             print(f"The following Firstrade accounts were found: {account_list}")
             print("Logged in to Firstrade!")
             firstrade_obj.set_logged_in_object(name, firstrade)
-            for i, account in enumerate(account_list):
-                firstrade_obj.set_account_number(name, account)
+            for i, entry in enumerate(account_info.all_accounts):
+                account = list(entry.keys())
+                firstrade_obj.set_account_number(name, account[0])
                 firstrade_obj.set_account_totals(
-                    name, account, str(account_info.account_balances[i])
+                    name, account[0], str(account_info.all_accounts[i][account[0]]['Balance'])
```

Comment:
According to Line 44, `account_info.all_accounts[i]` is equal to `entry` I think

Reply:
entry is a dictionary so it will not allow me to use it as an indices